apps/users/emails.py

```diff
@@ -47,7 +48,12 @@ def get_attachments(self):
         organisation = self.get_organisation()
         if organisation and organisation.logo:
             f = open(organisation.logo.path, "rb")
-            logo = MIMEImage(f.read())
```

Comment:
just assign the logo to None here, as python doesn't like calling variables not set outside conditional statements (we saw this bug in allauth)